packages/backend/src/graphql/resolvers/liquidity.ts

```diff
@@ -177,7 +178,7 @@ export const createPeerLiquidityWithdrawal: MutationResolvers<ApolloContext>['cr
       })
     }
     const peerService = await ctx.container.use('peerService')
-    const peer = await peerService.get(peerId)
+    const peer = await peerService.get(peerId, ctx.tenant.id)
```

Comment:
```suggestion
    const peer = await peerService.get(peerId, ctx.isOperator ? undefined : ctx.tenant.id)
```
so the operator can create liquidity withdrawals for any peer